homeassistant/components/uvc/config_flow.py

```diff
@@ -0,0 +1,42 @@
+"""Config flow for ZHA."""
+from collections import OrderedDict
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from . import CONF_KEY, CONF_NVR, CONF_PORT, CONF_SSL, DEFAULT_PORT, DEFAULT_SSL, DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class UvcFlowHandler(config_entries.ConfigFlow):
```

Comment:
Pass the domain as domain keyword argument and remove the decorator.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-your-config-flow